File: help/eng_gb/exercicios_da_ficha_7/Ficha07_001/Ficha07_001.cs
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

using System.IO;
using System;
public class Ficha07_001 {

    //Main Function
    static void Main() {
        Console.Write("Summation\n" );
        int sum = 0 ;
        for( int i = 1; i < 6; i = i + 1 ) {
            Console.Write(sum + " + " + i );
            sum++;
            Console.Write(" \t = " + sum + "\n" );
        }
        Console.Write("\nTotal = " + sum );
    }

}
//C#  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Fri, 2022 Sep 30 22:36

File: help/eng_gb/exercicios_da_ficha_7/Ficha07_001/Ficha07_001.cpp
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

#include <iostream>
#include <cstring>
#include <cmath>
using namespace std;

// Class definition
class Ficha07_001 {

    public:
        int main() {
            cout << "Summation\n";
            int sum  = 0 ;
            int i  = 0 ;
            for( int i = 1; i < 6; i = i + 1 ) {
                cout << sum  << " + " << i ;
                sum++;
                cout << " \t = " << sum  << "\n";
            }
            cout << "\nTotal = " << sum ;
            return 0;
        }
};// end of class Ficha07_001
// Execute main function
int main(){
    Ficha07_001 application; // create object
    application.main(); // execute object
    return 0;
}

//C++  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Fri, 2022 Sep 30 22:36
